src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1280,13 +1284,37 @@ class ContentExplorer extends Component<Props, State> {
         this.setState({ currentOffset: newOffset }, this.refreshCollection);
     };
 
+    /**
+     * Get the current viewMode, checking local store if applicable
+     *
+     * @return {ViewMode}
+     */
     getViewMode = (): ViewMode => {
         const { features }: Props = this.props;
         const viewModePreference = this.store.getItem(localStoreViewMode);
         const isGridViewEnabled = isFeatureEnabled(features, 'contentExplorer.gridView.enabled');
         return isGridViewEnabled && viewModePreference ? viewModePreference : VIEW_MODE_LIST;
     };
 
+    /**
+     * Get the maximum number of grid view columns based on the current width of the
+     * content explorer.
+     *
+     * @return {number}
+     */
+    getMaxNumberOfGridViewColumnsForWidth = (): number => {
```

Comment:
Can we group this with the `getGridColumnCount` helper? Maybe just `getGridColumnMax`?

Reply:
The result of `getMaxNumberOfGridViewColumnsForWidth` is used in `GridViewSlider`, while the result of `getContentGridColumnCount` is used in `Content`, so I don't think they can be grouped into one.  I'll refactor the code though to eliminate `getContentGridColumnCount`.